src/internal/focus-visible/index.ts

```diff
@@ -1,19 +1,28 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import { useEffect } from 'react';
+import React, { useEffect } from 'react';
 import { isModifierKey } from '../keycode';
 
+const frames = new Map<Document, { componentsCount: number; abortController: AbortController }>();
+
 function setIsKeyboard(active: boolean) {
   if (active) {
+    // For backwards compatibility if a ref isn't provided to useFocusVisible();
```

Comment:
What does this comment mean?

Reply:
Update: when ref is not provided, we do: `const currentDocument = componentRef?.current?.ownerDocument ?? document;`. Is the fallback still needed?